content/en/_index.md

```diff
@@ -831,6 +755,80 @@ $dlp = new DlpServiceClient();
 {{< /tab >}}
 {{< /tabpane >}}
 
+#### Authentication {#authentication}
+
+Samples should always authenticate using [Application Default Credentials][ADC].
+Most clients do this automatically, and no special steps are required.
+
+If, for whatever reason, the code snippet requires specific credentials,
+then, apart from showing how to obtain such credentials, it should show
+how to intialize a client with other than ADC.
+
+[ADC]:https://cloud.google.com/docs/authentication/external/about-adc
+
+{{< tabpane langEqualsHeader=true >}}
+{{< tab header="Java" >}}
+// Most clients use ADC by default. However, if your application needs to showcase a specific
+// credential source, show users how to do that explicitly.
+GoogleCredentials credentials = GoogleCredentials.fromStream(new FileInputStream("/path/to/credentials.json"));
+{{< /tab >}}
+{{< tab header="Go" >}}
+// Most clients use ADC by default. However, if your application needs to
+// showcase a specific Go credential source, show users how to do that
+// explicitly.
+ctx := context.Background()
+foo.NewClient(ctx, option.WithCredentialsFile("/path/to/credentials.json"))
+{{< /tab >}}
+{{< tab header="Python" >}}
+# Most clients use ADC by default. However, if your application needs to
+# showcase a specific credential source, show users how to do that explicitly.
+from google.oauth2 import service_account
+
+credentials = service_account.Credentials.from_service_account_file(
+  "/path/to/credentials.json"
+)
+{{< /tab >}}
+{{< tab header="Node.js" >}}
+// Most clients use ADC by default. However, if your application needs to showcase a specific
+// credential source, show users how to do that explicitly.
+const keyFile = '/path/to/credentials.json';
+const auth = new GoogleAuth({
+  keyFile: keyFile,
+  scopes: 'https://www.googleapis.com/auth/cloud-platform',
+});
+{{< /tab >}}
+{{< tab header="C#" >}}
+// All clients use ADC by default. However, if your application needs to use specific
+// credentials, show users how to initialize a client with other than ADC.
+GoogleCredentials specificCredentials = // show how to obtain these
```

Comment:
Given this isn't a sample but something to illustrate a practice, and this is not the preferred practice, maybe we should simply link to documentation about using service account keys instead of showing code at all?

Reply:
I'm suggesting maybe we should link to docs/samples on working with these credentials rather than show how to shape a sample for the edge case.

Reply:
I think language tabs slightly edge out creating a vertical list of languages, but I defer to your judgement.